core/trino-main/src/main/java/io/trino/server/ServerMainModule.java

```diff
@@ -408,11 +393,9 @@ protected void setup(Binder binder)
 
         // server info resource
         jaxrsBinder(binder).bind(ServerInfoResource.class);
-        jsonCodecBinder(binder).bindJsonCodec(ServerInfo.class);
```

Comment:
I think we can also remove jsoncodec binding for `Queryinfo`, `Selectedrole`,`QueryResults`.

Reply:
Thanks, I missed those